scripts/g.extension/testsuite/test_addons_download.py

```diff
@@ -0,0 +1,141 @@
+        """Test installing extension from github"""
+
+        self.assertModule(
+            "g.extension",
+            extension="r.example.plus",
+            url="https://github.com/wenzeslaus/r.example.plus",
+            prefix=str(self.install_prefix),
+        )
+
+        # Modules with non-standard branch would be good for testing...
+        self.assertModule(
+            "g.extension",
+            extension="r.example.plus",
+            url="https://github.com/wenzeslaus/r.example.plus",
+            prefix=str(self.install_prefix),
+        )
+
+        for file in self.files:
+            self.assertFileExists(file)
+
+    @unittest.skipIf(ms_windows, "currently not supported on MS Windows")
+    def test_gitlab_install(self):
+        """Test installing extension from gitlab"""
+        self.assertModule(
+            "g.extension",
+            extension="r.example.plus",
+            url="https://gitlab.com/vpetras/r.example.plus",
+            prefix=str(self.install_prefix),
+        )
+
+        for file in self.files:
+            self.assertFileExists(file)
+
+    @unittest.skipIf(ms_windows, "currently not supported on MS Windows")
+    def test_bitbucket_install(self):
+        """Test installing extension from bitbucket"""
+        files = [
+            self.install_prefix.joinpath("scripts", "r.sim.stats"),
+            self.install_prefix.joinpath("docs", "html", "r.sim.stats.html"),
+        ]
+        self.assertModule(
+            "g.extension",
+            extension="r.sim.stats",
+            url="https://bitbucket.org/lrntct/r.sim.stats",
+            prefix=str(self.install_prefix),
+        )
+
+        for file in files:
+            self.assertFileExists(file)
+
+    def test_github_install_official(self):
+        """Test installing C-extension from official addons repository"""
+        files = [
+            self.install_prefix.joinpath("bin", "r.gdd"),
+            self.install_prefix.joinpath("docs", "html", "r.gdd.html"),
+        ]
+        self.assertModule(
+            "g.extension", extension="r.gdd", prefix=str(self.install_prefix)
+        )
+
+        for file in files:
+            self.assertFileExists(file)
+
+    def test_github_install_official_multimodule(self):
+        """Test installing multi-module extension from official addons repository"""
+        files = [
+            self.install_prefix.joinpath("scripts", "i.sentinel.parallel.download"),
+            self.install_prefix.joinpath(
+                "docs", "html", "i.sentinel.parallel.download.html"
+            ),
+            self.install_prefix.joinpath("scripts", "i.sentinel.import"),
+            self.install_prefix.joinpath("docs", "html", "i.sentinel.import.html"),
+        ]
+        self.assertModule(
+            "g.extension", extension="i.sentinel", prefix=str(self.install_prefix)
+        )
+
+        for file in files:
+            self.assertFileExists(file)
+            if not str(file).endswith("html"):
```

Comment:
This should work the same and is more explicit in what the goal is:

```suggestion
            if not file.suffix == "html":
```